package.json

```diff
@@ -11,6 +11,7 @@
     "url": "https://github.com/RUGSoftEng/2017-Winter-Summer-School-App"
   },
   "dependencies": {
+    "angular-ui-bootstrap": "^2.5.6",
```

Comment:
Why has this been added? I do not see it referenced anywhere

Reply:
This was required for the pagination of forumpage.